### PR TITLE
Fetch Auto-execute setting from manager

### PR DESCRIPTION
**Pull request by jackie19**
Fetch Auto-execute setting from manager when reloading the browser

### CODE REVIEW

File: src/components/settings/SettingsStore.tsx [hunk 1]
```diff
@@ -144,6 +144,7 @@ export const useSettingsStore = create<SettingsState>((set, get) => ({
 				const fileContent = JSON.parse(file.content);
 
 				set({
+					autoExecuteGeneratedCode: fileContent.autoExecuteGeneratedCode,
```
Comment (alishobeiri) [inline src/components/settings/SettingsStore.tsx:147]: This should have a default value of `false`

Comment (alishobeiri) [reply src/components/settings/SettingsStore.tsx:147]: Or I guess to say it in different terms, please specify a default value, ie:
```
fileContent.autoExecuteGeneratedCode || false
```

**Comment by alishobeiri** [comment]
Think this was addressed in the latest commits. Thank you for opening a PR. Feel free to lose if you think it has already been addressed

**Comment by jackie19** [comment]
thanks

**Pull request closed without merging**